### PR TITLE
soc: x1e80100: Updates for 6.12

### PR DESCRIPTION
**Pull request by abelvesa**

### CODE REVIEW

File: _soc/x1e80100.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ status-debug-dcc:
 status-debug-eud:
 status-display-dp: "6.10"
 status-display-dsi: N/A
-status-display-hdmi: N/A
+status-display-hdmi: WIP
```
Comment (lumag) [inline _soc/x1e80100.md]: Huh? Is there an actual HDMI support on the chip? DP++?

Comment (abelvesa) [reply _soc/x1e80100.md]: No, it's DP-to-HDMI converter. So I guess I should drop this change then. My bad.

Comment (abelvesa) [reply _soc/x1e80100.md]: and it's off-chip (Realtek RTD2171) and it's only found on T14s, AFAICT.

Comment (lumag) [reply _soc/x1e80100.md]: Thanks. This is for SoC features only.

**Comment by lumag** [comment]
Also, could you please update all SoC statuses for 6.11 -> 6.12 cycle?

**Comment by abelvesa** [comment]
> Also, could you please update all SoC statuses for 6.11 -> 6.12 cycle?

Will do that starting tomorrow. Until then lets merge this for X Elite.

**Pull request merged**